build.sbt

```diff
@@ -2,7 +2,7 @@ import com.typesafe.sbt.packager.docker.Cmd
 
 name := """Drops"""
 
-version := "0.21.16"
+version := "0.21.17-test"
```

Comment:
You don't need to do this, since you can't know if somebody else is currently doing the same ;)